test/google_crawler_web/features/upload/view_new_upload_page_test.exs

```diff
@@ -0,0 +1,10 @@
+defmodule GoogleCrawlerWeb.Upload.ViewNewUploadPageTest do
+  use GoogleCrawlerWeb.FeatureCase
+
+  feature "view new upload page", %{session: session} do
```

Comment:
```suggestion
  feature "views new upload page", %{session: session} do
```

Reply:
Fixed in https://github.com/ankitkalia1195/elixir-google-crawler/pull/34/commits/f300517565b4673169c7ca8270285d66e2c1cbfe